services/hanke-service/src/main/kotlin/fi/hel/haitaton/hanke/permissions/Permissions.kt

```diff
@@ -11,10 +11,20 @@ import jakarta.persistence.Id
 import jakarta.persistence.JoinColumn
 import jakarta.persistence.ManyToOne
 import jakarta.persistence.Table
+import java.util.*
```

Comment:
Some kind of automatic import from IDE?

Reply:
It seems that this keeps happening with UUID imports no matter what I do. Fixed, thx.